backend/layers/processing/process_download.py

```diff
@@ -89,21 +83,19 @@ def get_job_definion_name(dataset_version_id: str) -> str:
     @staticmethod
     def estimate_resource_requirements(
         adata: scanpy.AnnData,
-        memory_modifier: float = MEMORY_MODIFIER,
-        min_memory_MB: int = MIN_MEMORY_MB,
-        max_memory_MB: int = MAX_MEMORY_MB,
-        max_vcpu: int = MAX_VCPU,
-        max_swap_memory_MB: int = MAX_SWAP_MEMORY_MB,
-        swap_modifier: int = SWAP_MODIFIER,
-        memory_per_vcpu: int = MEMORY_PER_VCPU,
+        memory_modifier: float = config.ingest_memory_modifier,
+        min_vcpu: int = config.ingest_min_vcpu,
+        max_vcpu: int = config.ingest_max_vcpu,
+        max_swap_memory_MB: int = config.ingest_max_swap_memory_mb,
+        swap_modifier: int = config.ingest_swap_modifier,
+        memory_per_vcpu: int = 8000,
```

Comment:
any reason this one is still set here rather than the config?

Reply:
because this is based on what AWS does and not based on our config.  AWS allocates 8GB per VCPU so we are following that pattern in our estimation.

Reply:
gotcha, thanks